app/src/types/user.js

```diff
@@ -42,13 +44,17 @@ export type User = t.TypeOf<typeof UserModel>;
  * @param from
  * @returns {User}
  */
-export function extractUserFromRequest(from: express$Request): User {
+export function extractUserFromRequest(from: express$Request): Option<User> {
   const reqWithUser = ((from: Object): { user: User });
 
-  // eslint-disable-next-line no-unused-vars
-  return t.validate(reqWithUser.user, UserModel).fold(_ => {
-    return createEmptyUser();
-  }, t.identity);
+  return t.validate(reqWithUser.user, UserModel).fold(
```

Comment:
`validate` returns an [Either](https://github.com/gcanti/fp-ts/blob/master/src/Either.ts) - `Either` has a `toOption` method that does exactly this